lib/praxis/tasks/api_docs.rb

```diff
@@ -45,6 +45,15 @@
       generator = Praxis::RestfulDocGenerator.new(Dir.pwd)
     end
 
+    desc "Generate EXPERIMENTAL API docs (JSON definitions) for a Praxis App"
+    task :experiments => [:environment] do |t, args|
```

Comment:
let's change it and call it something a little more appropriate...for example: :generate_beta?

i.e., praxis:docs:generate_beta